packages/checkout/utils/validation/isPostcode.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { POSTCODE_REGEXES } from 'postcode-validator/lib/cjs/postcode-regexes.js';
+
+const getCustomRegexes = () => {
```

Comment:
Can you explain why this is a function? It looks to just be changing `POSTCODE_REGEXES`,

Reply:
I thick it's just a different style of doing things, you can also end up merging the 2 arrays.

However I go agree, this override log will run each time we run validation, which is too much, move it to a static variable set.

Reply:
> However I go agree, this override log will run each time we run validation, which is too much, move it to a static variable set.

Thanks for suggesting this, Nadir. As requested, I just converted the function to a static variable set.